taskfiles/utils-remote.yaml

```diff
@@ -77,6 +77,8 @@ tasks:
         ref: "default (list) .INCLUDE_PATTERNS"
 
       # Misc
+      ARCHIVER: >-
+          {{if eq OS "darwin"}}gtar{{else}}tar{{end}}
```

Comment:
Why do we want to use `gtar` on macOS?

Reply:
The `tar` command doesn't support `--directory`, `--no-anchored`, or `--wildcards`. However, it does provide the shorthand `-C` option as an alternative to `--directory`, but the other two options are not available in the default version of `tar`.

Reply:
References:
`gtar`: https://linux.die.net/man/1/gtar
`tar`: https://ss64.com/mac/tar.html

Reply:
Can you add that to the PR description?

Reply:
ok

Reply:
added